tap_bing_ads/reports.py

```diff
@@ -27,10 +27,7 @@
         'AdExtensionId',
         'AdExtensionPropertyValue',
         'AdExtensionType',
-        'AdExtensionTypeId',
-        'Clicks',
-        'Ctr',
-        'Impressions'
+        'AdExtensionTypeId'
```

Comment:
@namrata270998 Can you please add a comment regarding the change you have made in this PR.
Also add a code comment above this line regarding what these fields are meant for

Reply:
Bing has a requirement that each report must have at least 1 performance based metric (Clicks, Impressions, etc.) selected to generate a report. We force `Impressions`, `Ctr,`, and `Clicks` to be automatic for the `ad_extension_detail_report`. This is inconsistent with how we treat field selection for other reports as we do not impose any performance measures on any other report streams. It should be up to user, which perf measures should be selected. Hence, removed the `Impressions`, `Ctr,`, and `Clicks` from required fields.